metricbeat/Makefile

```diff
@@ -73,7 +73,7 @@ test-module: python-env update metricbeat.test
 	. ${PYTHON_ENV}/bin/activate && INTEGRATION_TESTS=1 nosetests tests/system/test_${MODULE}.py
 
 .PHONY: assets
-assets: libbeat_fields
```

Comment:
Unfortunately I don't think Metricbeat is free from this dependency yet due to the way that the fields.go files are composed.

One solution would be to include both metricbeat common fields and libbeat fields in `metricbeat/include/fields.go`. This could be generated by running `global_fields` without passing in the `module` dir as an argument.

It's be nice if we could pipe the output of one command into another like:

`go run ${ES_BEATS}/libbeat/scripts/cmd/global_fields/main.go -es_beats_path ${ES_BEATS} -beat_path ${BEAT_NAME} | go run ${ES_BEATS}/dev-tools/cmd/asset/asset.go -pkg include -in - -out include/fields.go`

Reply:
Let's hope this becomes obsolete again in the near future :-)